src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -107,7 +107,7 @@ final void classBegin() {
         if (superName != null) {
             extendsExpr = " extends " + superName;
         }
-        appendLines(String.format("%s %s %s%s {", modifiers, kind.kindName, className, extendsExpr));
+        appendLines("%s %s %s%s {", modifiers, kind.kindName, className, extendsExpr);
```

Comment:
no numbers here?

Reply:
Fixed in [8507b05](https://github.com/openjdk/jextract/pull/244/commits/8507b0533a46d54cfbd8b040aeddcead3aa70e6d), thanks